style/rails/migration.rb

```diff
@@ -1,9 +1,10 @@
 class CreateClearanceUsers < ActiveRecord::Migration
   def change
     create_table :users  do |t|
-      t.timestamps null: false
       t.string :email, null: false
       t.string :name, null: false, default: ''
+
+      t.timestamps null: false
```

Comment:
If it's removed, it can look like we advocate no timestamps, which is more problematic than its position.

How about remove the guideline but leave the timestamps in the example at the top?